install_config/configuring_pipeline_execution.adoc

```diff
@@ -7,26 +7,32 @@
 :experimental:
 :toc: macro
 :toc-title:
+:prewrap!:
 
 toc::[]
 
 
 == Overview
 
-{product-title} utilizes Jenkins to execute continuous development pipelines.  To
-provide a seamless experience for users, the first time a user creates a build
-configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
-strategy type, a set of Jenkins related resources will be created within the user's project.
-By default, the system will look for a template named `*jenkins*` in the `*openshift*` namespace and
-instantiate it within the user's project.  {product-title} ships with a default template definition
-which will, upon instantiation, create the following in the user's project:
+{product-title} utilizes Jenkins to execute continuous development pipelines.
+To provide a seamless experience for users,
+the first time a user creates a build configuration using the
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[Pipeline]
+strategy type,
+a set of Jenkins related resources will be created within the user's project.
+By default, the system will look for a template named `*jenkins*`
+in the `*openshift*` namespace and instantiate it within the user's project.
+{product-title} ships with a default template definition which will,
```

Comment:
@bparees With `oc get template -n openshift`, i see:

```
NAME                     DESCRIPTION                               ...
...
jenkins-ephemeral        Jenkins service, without persistent storage.
jenkins-persistent       Jenkins service, with persistent storage.
...
```

which i believe points to [these files](https://github.com/openshift/origin/tree/master/examples/jenkins).  (I looked at, via `oc get template`, both templates and their content is largely the same as the similarly named files.)  So, my question above is now obsoleted by these others:

- Are these files the actual source for the jenkins-{ephemeral,persistent} templates?
  * If so, are they visible to the cluster administrator (installed somewhere)?
  * If not, where can i find the original source?
- Which (ephemeral, persistent) is the "default" mentioned in the docs?
- If the answer to that is "it depends", what does it depend on and when is that determination made?